can/interfaces/pcan/pcan.py

```diff
@@ -620,16 +620,19 @@ def _detect_available_configs():
                 }
             )
         for i in interfaces:
-            error, value = library_handle.GetValue(i["id"], PCAN_CHANNEL_CONDITION)
-            if error != PCAN_ERROR_OK or value != PCAN_CHANNEL_AVAILABLE:
-                continue
-            has_fd = False
-            error, value = library_handle.GetValue(i["id"], PCAN_CHANNEL_FEATURES)
-            if error == PCAN_ERROR_OK:
-                has_fd = bool(value & FEATURE_FD_CAPABLE)
-            channels.append(
-                {"interface": "pcan", "channel": i["name"], "supports_fd": has_fd}
-            )
+            try:
+                error, value = library_handle.GetValue(i["id"], PCAN_CHANNEL_CONDITION)
+                if error != PCAN_ERROR_OK or value != PCAN_CHANNEL_AVAILABLE:
+                    continue
+                has_fd = False
+                error, value = library_handle.GetValue(i["id"], PCAN_CHANNEL_FEATURES)
+                if error == PCAN_ERROR_OK:
+                    has_fd = bool(value & FEATURE_FD_CAPABLE)
+                channels.append(
+                    {"interface": "pcan", "channel": i["name"], "supports_fd": has_fd}
+                )
+            except AttributeError:  # Ignore if this fails for some interfaces
+                pass
```

Comment:
Do we know the reason? Is some warning necessary here?

Reply:
No, I do not really know what is failing here.

Reply:
Let's just open an issue on this.

Reply:
Tracked in #1007